CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/DatabaseConfigurationRequestHandler.swift

```diff
@@ -41,23 +43,26 @@ public class DatabaseConfigurationRequestHandler {
             let config: DatabaseConfiguration = args.get(name: "config")!
             return config.directory
 
+        #if COUCHBASE_ENTERPRISE
         case "databaseConfiguration_getEncryptionKey":
             let config: DatabaseConfiguration = args.get(name: "config")!
             return config.encryptionKey
-
+        #endif
         case "databaseConfiguration_setDirectory":
             let config: DatabaseConfiguration = args.get(name: "config")!
             let directory: String = args.get(name: "directory")!
             config.directory = directory
             return config
-
+        
+        #if COUCHBASE_ENTERPRISE
         case "databaseConfiguration_setEncryptionKey":
```

Comment:
move this case along with getEncryptionKey case so that we can use only single if endif clause

Reply:
done

Reply:
COUCHBASE_ENTERPRISE must be a project level flag, and by default it's used this way. But mixing with case statement looks a little confusing. If there is better solutions, it will be great, otherwise, I am ok with it